openmdao/drivers/genetic_algorithm_driver.py

```diff
@@ -221,6 +221,9 @@ def run(self):
         Pm = self.options['Pm']  # if None, it will be calculated in execute_ga()
         Pc = self.options['Pc']
 
+        # Check for missing objective
```

Comment:
this comment is unnecessary.  The function name very clearly indicates what's happening.